code_fix_floater/flt_init_varia.F

```diff
@@ -119,12 +119,19 @@ SUBROUTINE FLT_INIT_VARIA ( myThid  )
 
       IF ( ioUnit.GT.0 .AND. mapIniPos2Index ) THEN
 C--   Found a global file
-        WRITE(msgBuf,'(A,2I4,A,1P2E15.8)')
-     &    ' bi,bj=', bi, bj, ' , npart,max_npart=', tmp(1), tmp(6)
+C-------Start Nathaniel Hack----------------------
+        IF (flt_npart.EQ.0 ) THEN
+          npart_read = NINT(tmp(1))
+        ELSE
+          npart_read = flt_npart
+        ENDIF
```

Comment:
I like this...if you leave out flt_npart from data.flt, it still tries get it from the file.